Cargo.toml

```diff
@@ -141,3 +141,7 @@ harness = false
 name = "cratesfyi"
 test = false
 doc = false
+
+[patch.crates-io.onig_sys]
+# Once new version (> 69.8.1) is released, we can remove this patch.
+git = "https://github.com/rust-onig/rust-onig"
```

Comment:
I really don't like using the master branch here. 

IMO it's (a little) better if we either use a fixed git hash, or even use just the fixing commit as patch that's needed (if that's possible).

Reply:
Indeed, a hash would avoid further issues.